### DISCUSSION TITLE
Dexcom One+ restart

### DISCUSSION THREAD

**Discussion by CielavaG** [NightscoutFoundation/xDrip#3494] [Q&A]
The Dexcom One sensor could be reused for an additional 10 days when using the XDrip. Can the One+ sensor also be restarted by XDrip, extending its life?

**Comment by Navid200** [comment]
xDrip cannot extend the life of Dexcom One or One+.

Under certain circumstances, you may have no choice but to restart a G6 or Dexcom One sensor.
There is no way to restart a G7 or One+.
None of this has anything to do with xDrip.

**Reply by CielavaG** [reply]
Okay, maybe I said it wrong. The Dexcom One can take the transmitter out, put it back in, and use the same sensor again. XDrip is used to stop the sensor before removing it and then restart it again.
But the One+ sensor is already included with the transmitter. It turns out that it will not be possible to save money.
